src/lib/blkalloc/append_blk_allocator.h

```diff
@@ -0,0 +1,103 @@
+/*********************************************************************************
+ * Modifications Copyright 2017-2019 eBay Inc.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * *
+ * *********************************************************************************/
+#pragma once
+
+#include <sisl/logging/logging.h>
+#include <homestore/blk.h>
+#include "blk_allocator.h"
+#include "common/homestore_assert.hpp"
+#include "common/homestore_config.hpp"
+#include <homestore/superblk_handler.hpp>
+
+namespace homestore {
+static constexpr uint64_t append_blkalloc_sb_magic{0xd0d0d02b};
```

Comment:
Some interesting magic here, does it convey anything :-)

Reply:
just don't want to be same as other magic, I think I found similar pattern otherwhere in homestore, maybe logstore magic? :)